backend/src/test/java/de/neuefische/finalproject/ohboy/controller/MonsterControllerTest.java

```diff
@@ -0,0 +1,165 @@
+package de.neuefische.finalproject.ohboy.controller;
+
+import de.neuefische.finalproject.ohboy.dao.MonsterMongoDao;
+import de.neuefische.finalproject.ohboy.dao.UserDao;
+import de.neuefische.finalproject.ohboy.dto.AddMonsterDto;
+import de.neuefische.finalproject.ohboy.dto.FacebookCodeDto;
+import de.neuefische.finalproject.ohboy.dto.FacebookUserDto;
+import de.neuefische.finalproject.ohboy.model.Monster;
+import de.neuefische.finalproject.ohboy.model.OhBoyUser;
+import de.neuefische.finalproject.ohboy.service.FacebookApiService;
+import de.neuefische.finalproject.ohboy.utils.IdUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.*;
+import org.springframework.test.context.TestPropertySource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(properties = {
+        "jwt.secretkey=somesecrettoken",
```

Comment:
Doesn't change anything, but this is not a token, the value might be a little misleading.